### ISSUE TITLE
fix Numbered list bugs

### ISSUE THREAD

**Issue by Fahad-Alsaidi** [HOST-Oman/scribus#244]
Here is a list of bugs needs to be fixed to get numbered list works fine:

- [ ] in some cases delete button doesn't delete numbered list.
- [ ] add indentation option to intend the numbered list, this good for sub-levels.
- [ ] make levels work in sane way
- [ ] fix prefix & suffix specially in sublevels.
- [ ] make levels's separator customizable 
- [ ] fix update numbered list styles.

@aoloe @ftake please mention here any bugs related to numbered list

**Comment by aoloe** [comment]
sadly, i don't use (yet) numbered lists...

in the layouts i do, i don't have many lists and when i have some i go for the old n-dash and tabulator way...

just one important (i think) note: the lists code has been written by cezary at a time he was not an experienced programmer.

i've used his code for the table of contents as an inspiration, but i had to basically rewrite everything, since his code was really hard to follow / maintain.

this code has made it into the trunk, so it's probably in a better shape, but i would suggest you not to hesitate rewriting parts of the code, if you think it helps avoiding bugs!

**Comment by gyuris** [comment]
Dear @Fahad-Alsaidi, please look at my footnote mockup here: https://bugs.scribus.net/view.php?id=14588 I suggest "Before" (Nothing, Tab) and "After" (Space, Tab, [New Line](https://bugs.documentfoundation.org/show_bug.cgi?id=112384)) property for numbering to.
